src/pyflakes/snapshots/ruff__pyflakes__tests__F831_F831.py.snap

```diff
@@ -2,31 +2,5 @@
 source: src/pyflakes/mod.rs
 expression: checks
 ---
-- kind: DuplicateArgumentName
-  location:
-    row: 1
-    column: 24
-  end_location:
-    row: 1
-    column: 30
-  fix: ~
-  parent: ~
-- kind: DuplicateArgumentName
-  location:
-    row: 5
-    column: 27
-  end_location:
-    row: 5
-    column: 33
-  fix: ~
-  parent: ~
-- kind: DuplicateArgumentName
-  location:
-    row: 9
-    column: 26
-  end_location:
-    row: 9
-    column: 32
-  fix: ~
-  parent: ~
+[]
```

Comment:
`def f(a, a): pass ` now yields `SyntaxError` ( https://github.com/RustPython/RustPython/pull/4380).

Reply:
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/17039389/210124730-2b25cdd2-d849-4fc3-988d-1786646379cc.png">

Reply:
Maybe we should remove F831.

Reply:
or we can just keep it as it's not harmful.

Reply:
Ah yeah, we can remove in a separate PR.

(Is this true in all Python versions 3.7+? If you know.)

Reply:
@charliermarsh 

```
> docker run python:3.7 python -c "def f(a, a): pass"
  File "<string>", line 1
SyntaxError: duplicate argument 'a' in function definition

> docker run python:3.8 python -c "def f(a, a): pass"
  File "<string>", line 1
SyntaxError: duplicate argument 'a' in function definition

> docker run python:3.9 python -c "def f(a, a): pass"
  File "<string>", line 1
SyntaxError: duplicate argument 'a' in function definition

> docker run python:3.10 python -c "def f(a, a): pass"
  File "<string>", line 1
SyntaxError: duplicate argument 'a' in function definition
```

Reply:
@charliermarsh 

> we can remove in a separate PR.

I'll file a PR.